workflow/validate/validate.go

```diff
@@ -295,7 +296,10 @@ func ValidateClusterWorkflowTemplate(wftmplGetter templateresolution.WorkflowTem
 }
 
 // ValidateCronWorkflow validates a CronWorkflow
-func ValidateCronWorkflow(wftmplGetter templateresolution.WorkflowTemplateNamespacedGetter, cwftmplGetter templateresolution.ClusterWorkflowTemplateGetter, cronWf *wfv1.CronWorkflow) error {
```

Comment:
Could this method be left as-is and a new one be introduced? #2793 is using this

Reply:
Sure, I see what you mean. I think ideally we make the change in https://github.com/argoproj/argo/pull/3209#discussion_r439489971 and just passed in the merged getter